.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      github-action-dependencies:
+        patterns:
+          - "*"
+
+  - package-ecosystem: "npm"
+    directory: "/M2/Macaulay2/editors/prism"
+    schedule:
+      interval: "monthly"
+    groups:
+      prism-dependencies:
+        patterns:
+          - "*"
+
+  - package-ecosystem: "npm"
+    directory: "/M2/Macaulay2/editors/highlightjs"
+    schedule:
+      interval: "monthly"
+    groups:
+      highlightjs-dependencies:
+        patterns:
+          - "*"
+
+  - package-ecosystem: "npm"
+    directory: "/M2/Macaulay2/packages/Style"
+    schedule:
+      interval: "monthly"
+    groups:
```

Comment:
Will this group dependency updates for prism and highlightjs into one PR? That's what I had in mind, based on the first example [here](https://github.blog/2023-08-24-a-faster-way-to-manage-version-updates-with-dependabot/#example-1-group-by-dependency-type), but I'm not sure.

Reply:
Currently, prism and highlightjs get separate PR's.  I'll play around with it some more and see if we can combine them.

Reply:
It looks like it's one group per element of the "updates" list in yaml file (so prism and highlightjs are separate).  I tried moving the "groups" key up a level and I got an error: "The property '#/' contains additional properties ["groups"] outside of the schema when none are allowed"